INSTALL.md

```diff
@@ -81,7 +81,7 @@ For debugging, each integration test can be run individually and with various co
 
 ### Doxygen
 
-A webpage base on [Doxygen](http://www.doxygen.nl) is generated by running `make doc` (if doxygen is installed).
+A webpage baseed on [Doxygen](http://www.doxygen.nl) is generated by running `make doc` (if doxygen is installed).
```

Comment:
"based"

Reply:
Already done.

Reply:
"baseed" is not the correct spelling, it's going in the right direction, but one 'e' too far

Reply:
I did not see the extra 'e'!